packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -485,11 +490,12 @@ class DatePicker extends React.Component {
                 onButtonFocus={this.handleFocus}
                 buttonRefCallback={(buttonRef) => { this.calendarButton = buttonRef; }}
                 ariaLabel={ariaLabel}
+                inputRefCallback={inputRefCallback}
                 useExternalFormatMask={useExternalFormatMask}
                 initialTimeZone={initialTimeZone}
               />
             )}
-            customInputRef="inputRefCallback"
+            // customInputRef="inputRefCallback"
```

Comment:
Should this be removed or fixed?

Reply:
This was change was added as part of the react-intl upgrade.
Have fixed these now to incorporate the upgrade changes as well.